### DISCUSSION TITLE
Control Panel 404

### DISCUSSION THREAD

**Discussion by xmrtni** [statamic/cms#4521] [Q&A]
I've added Statamic to an existing Laravel app, and while everything seemed to install just fine (I can see all added files/folders) I get a 404 error when I try to go to the control panel (myapp:8080/cp). I also get the following error in the Laravel debugger: No query results for model [App\Models\Page].

All of the other resources for the cp, like css and images, all have a 200 status. I created a new Statamic project and copied over the htaccess file just in case, but no luck. I've searched online and found a few others that had the same problem, but they all had unique issues that don't apply to me. I'm developing locally using MAMP (apache). 

I've been stuck on this for a while, so any guidance in the right direction is appreciated!

Edit: I have Jetstream installed, I'm wondering if that is causing the issue somewhere.

**Comment by edalzell** [comment]
Do you have any wildcard routes that would be taking precedence?

**Reply by xmrtni** [reply]
No, no wildcards.

**Comment by jasonvarga** [comment]
What's the `App\Models\Page` model? What are you doing with that? Sounds like it's trying to be used when you hit /cp. Seems like you might have some routing logic set up already that is conflicting with the cp.
